packages/components/psammead-brand/src/index.jsx

```diff
@@ -66,30 +71,68 @@ const BrandSvg = styled.svg`
   /* stylelint-enable */
 `;
 
-const Brand = ({ brandName, svgHeight, minWidth, maxWidth, svg }) => (
-  <Banner svgHeight={svgHeight}>
+/* eslint-disable react/prop-types */
+const StyledBrand = ({ brandName, svgHeight, svg, maxWidth }) => (
```

Comment:
`StyledLink` is a styled-components `styled.a` element, but `StyledBrand` is a React element. Can we ensure consistancy, if we need to we should only prefix directly styled-components objects as `Styled`

Reply:
This has been resolved but its not changed?